### PR TITLE
Derandomize addresses.

### PR DESCRIPTION
**Pull request by dvc94ch**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -192,8 +198,10 @@ impl Network {
         self.range
     }
 
-    pub fn random_addr(&self) -> Ipv4Addr {
-        self.range.random_client_addr()
+    pub fn random_addr(&mut self) -> Ipv4Addr {
```
Comment (rklaehn) [inline src/lib.rs]: This is not really a random_addr anymore, right? You want to keep the name for compat?

Comment (dvc94ch) [reply src/lib.rs]: well it's more like "I don't care give me an addr". compat doesn't really matter as `&mut self` breaks stuff anyway. Better suggestions for a name?

Comment (wngr) [reply src/lib.rs]: `new_addr`?

Comment (dvc94ch) [reply src/lib.rs]: We settled on unique_addr in slack

**Pull request merged**